src/App.jsx

```diff
@@ -46,6 +48,12 @@ export class App extends React.Component {
         return <About />;
       case paths.CONTACT:
         return <Contact />;
+      case paths.EVENTS_APPEARANCES:
+        return <Events type={eventTypes.APPEARANCES} />;
+      case paths.EVENTS_DONATIONS:
+        return <Events type={eventTypes.DONATION} />;
+      case paths.EVENTS_QUOTES:
+        return <Events type={eventTypes.QUOTE} />;
```

Comment:
I think we can keep this, but I think we should first focus on getting the Quotes UX right, without focusing on the "See more" and dedicated Events page

Also, should we disable the `APPEARANCES` and `DONATIONS` types for now?